### PR TITLE
[doc] Add docs on JNI compilation

### PR DESCRIPTION
**Pull request by KexinFeng**

### CODE REVIEW

File: engines/pytorch/pytorch-native/README.md [hunk 1]
```diff
@@ -98,3 +98,54 @@ It uses clang-format to format the code.
...
 3. Publish to sonatype 
 4. Raise a PR to remove all the -SNAPSHOT
+
+## A quick start on implementing pytorch feature with JNI
+This is adjusted from the comments in [PR#2349](https://github.com/deepjavalibrary/djl/issues/2349#issuecomment-1409003379).
+
+To implement a simple pytorch feature, generally you can do the following steps.
+1. Find the c-api in torch library for the feature to add. This can be done by searching in the document like [this](https://pytorch.org/cppdocs/api/function_namespaceat_1a854b1b19549a17f87a69b5f6b1134e22.html?highlight=bmm) or searching in the torch cpp source code.
+2. Implement the JNI and api's in Java. The JNI can then be compiled with gradle commands. Here is the commands you can use on cpu machine, to compile JNI and run it with java api.
+```
```
Comment (frankfliu) [inline engines/pytorch/pytorch-native/README.md:108]: Add an empty line before ```

File: engines/pytorch/pytorch-native/README.md [hunk 2]
```diff
@@ -98,3 +98,54 @@ It uses clang-format to format the code.
...
+./gradlew :engines:pytorch:pytorch-native:cleanJNI ;\
+./gradlew :engines:pytorch:pytorch-native:compileJNI -Pcu11
+```
+3. Document the api and add the unit tests.
+
+4. Format the code and pass the PR tests
+
+Run the following tasks
+```
+./gradlew fJ fC checkstyleMain checkstyleTest pmdMain pmdTest ;\
```
Comment (frankfliu) [inline engines/pytorch/pytorch-native/README.md:147]:
```suggestion
./gradlew fJ fC checkstyleMain checkstyleTest pmdMain pmdTest
```

File: engines/pytorch/pytorch-native/README.md [hunk 3]
```diff
@@ -98,3 +98,54 @@ It uses clang-format to format the code.
...
+./gradlew verifyJava ;\
```
Comment (frankfliu) [inline engines/pytorch/pytorch-native/README.md:148]:
```suggestion
```

File: engines/pytorch/pytorch-native/README.md [hunk 4]
```diff
@@ -98,3 +98,54 @@ It uses clang-format to format the code.
...
 4. Raise a PR to remove all the -SNAPSHOT
+
+## A quick start on implementing pytorch feature with JNI
+This is adjusted from the comments in [PR#2349](https://github.com/deepjavalibrary/djl/issues/2349#issuecomment-1409003379).
+
+To implement a simple pytorch feature, generally you can do the following steps.
+1. Find the c-api in torch library for the feature to add. This can be done by searching in the document like [this](https://pytorch.org/cppdocs/api/function_namespaceat_1a854b1b19549a17f87a69b5f6b1134e22.html?highlight=bmm) or searching in the torch cpp source code.
+2. Implement the JNI and api's in Java. The JNI can then be compiled with gradle commands. Here is the commands you can use on cpu machine, to compile JNI and run it with java api.
+```
+./gradlew cleanJNI; \
```
Comment (frankfliu) [inline engines/pytorch/pytorch-native/README.md:109]: put all command in one line is not easy for read.

File: engines/pytorch/pytorch-native/README.md [hunk 5]
```diff
@@ -98,3 +98,54 @@ It uses clang-format to format the code.
...
+./gradlew :engines:pytorch:pytorch-jni:clean; \
+./gradlew :engines:pytorch:pytorch-native:clean; \
+./gradlew :engines:pytorch:pytorch-native:compileJNI; \
+\
+./gradlew :engines:pytorch:pytorch-jni:clean :engines:pytorch:pytorch-jni:build; \
```
Comment (frankfliu) [inline engines/pytorch/pytorch-native/README.md:114]:
```suggestion
```

File: engines/pytorch/pytorch-native/README.md [hunk 6]
```diff
@@ -98,3 +98,54 @@ It uses clang-format to format the code.
...
+./gradlew :engines:pytorch:pytorch-engine:clean :engines:pytorch:pytorch-engine:build;
```
Comment (frankfliu) [inline engines/pytorch/pytorch-native/README.md:115]:
```suggestion
./gradlew :engines:pytorch:pytorch-engine:build
```

**Review by lanking520 — APPROVED** [review]

**Pull request closed without merging**